daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -35,9 +35,8 @@ final case class EngineConfig(
     stackTraceMode: Boolean = false,
     profileDir: Option[Path] = None,
     contractKeyUniqueness: ContractKeyUniquenessMode = ContractKeyUniquenessMode.On,
-    // TODO: https://github.com/digital-asset/daml/issues/10504
-    //  switch the default to true
-    requireSuffixedGlobalCids: Boolean = false,
+    requireV1ContractId: Boolean = false,
+    requireSuffixedGlobalContractId: Boolean = false,
```

Comment:
did you drop the todo intentionally?

Reply:
Yes. After our discussion last week where you told me you are not concern about misusage of the flags (because we have only 3 actively maintain ledgers) I just thought the default behavior does not really matter.